src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -22,8 +22,19 @@
 
   <ItemGroup>
     <AdditionalRuntimeIdentifiers Include="$(AdditionalRuntimeIdentifiers)" Imports="$(AdditionalRuntimeIdentifierParent)" />
+  </ItemGroup>
+
+  <!-- Workaround for https://github.com/dotnet/sdk/pull/45566: Ensure that the OS portion of the RID we're building for is part of the RID graph as a separate RID. -->
+  <PropertyGroup Condition="'$(DotNetBuildSourceOnly)' == 'true'">
+    <OutputRidOS Condition="$(OutputRID.Contains('-'))">$(OutputRID.Split('-')[0])</OutputRidOS>
+    <AdditionalRidParentOS Condition="$(AdditionalRuntimeIdentifierParent.Contains('-'))">$(AdditionalRuntimeIdentifierParent.Split('-')[0])</AdditionalRidParentOS>
+    <AdditionalRidParentOS Condition="!$(AdditionalRuntimeIdentifierParent.Contains('-'))">$(AdditionalRuntimeIdentifierParent)</AdditionalRidParentOS>
+  </PropertyGroup>
```

Comment:
Is the entire property group the workaround? When will we be able to remove this?

Reply:
Is the L36 (with your changes) also temporary?